LikelihoodToRespond.md

```diff
@@ -0,0 +1,70 @@
+Likelihood To Respond Service
+=============
+
+Table of Contents
+_________
+- [Overview](#overview)
+- [Request Information](#request-information)
+- [Response Information](#response-information)
+- [Sample Request](#sample-request)
```

Comment:
oh, I meant to take this line out for now. I think what I'll do instead is just put an example query string for each result type in the Request Information section